### ISSUE TITLE
add a parameter to change security_driver for qemu.conf

### ISSUE THREAD

**Issue by rkhozinov** [thias/puppet-libvirt#68]
# The default security driver is SELinux. If SELinux is disabled                
# on the host, then the security driver will automatically disable              
# itself. If you wish to disable QEMU SELinux security driver while             
# leaving SELinux enabled for the host in general, then set this                
# to 'none' instead.                                                            
#                                                                               
# security_driver = "selinux"

**Comment by rkhozinov** [comment]
#69

**Comment by asasfu** [comment]
I agree, this is a desired attribute, until a PR gets built to manage the conf files via hash so that we can alter multiple values without having service order issues when we need to modify more lines; it'd be nice to see this PR get merged. @igalic @thias 
Thanks 😄
